test/karma/test-app/custom-elements-output-webpack/webpack.config.js

```diff
@@ -1,20 +0,0 @@
-const path = require('path');
-const { WWW_OUT_DIR } = require('../../constants');
-
-module.exports = {
-  entry: path.resolve(__dirname, 'index.esm.js'),
```

Comment:
Does someone know the point of having Webpack involved here? It doesn't seem to do anything other than re-bundling the package and importing it in the browser. I believe we cover this feature with the [setup script](https://github.com/ionic-team/stencil/blob/908f7780f5979beeb99a4b5f0aedb626590a8ec8/test/wdio/setup.ts) automatically.

Reply:
I think @rwaskiewicz is familiar with the basis for the setup -- if it's possible for us to do without it I'm all for it because it's super confusing haha

Reply:
IIRC, reason for this was two-fold:
1. to test Stencil could go through the webpack process (this is a regression Stencil had at some point)
2. To use the `dist-custom-elements` output target, as opposed to `www`